source/plugins/go/src/extension/extension_linux.go

```diff
@@ -17,8 +19,11 @@ func getExtensionConfigResponse(jsonBytes []byte) ([]byte, error) {
 
 	fs := &FluentSocket{}
 	fs.sockAddress = "/var/run/mdsd-ci/default_fluent.socket"
+	genevaLogsIntegrationEnabled := strings.TrimSpace(strings.ToLower(os.Getenv("GENEVA_LOGS_INTEGRATION")))
 	if containerType != "" && strings.Compare(strings.ToLower(containerType), "prometheussidecar") == 0 {
 		fs.sockAddress = fmt.Sprintf("/var/run/mdsd-%s/default_fluent.socket", containerType)
+	} else if genevaLogsIntegrationEnabled != "" && strings.Compare(strings.ToLower(genevaLogsIntegrationEnabled), "true") == 0 {
+		fs.sockAddress = "/var/run/mdsd-PrometheusSidecar/default_fluent.socket"

```

Comment:
Doesn't the above if also make sure the address is mdsd-prometheussidecar? Since we are checking that the containerType is prometheussidecar? We can combine the if and else right?